artifactory/commands/helm/repository.go

```diff
@@ -0,0 +1,129 @@
+package helm
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+const (
+	oci          = "oci://"
+	schemeHttp   = "https"
+	schemeSecure = "http"
+)
+
+// getHelmRepositoryFromArgs extracts repository name from helm push command registry URL
+func getHelmRepositoryFromArgs() (string, error) {
+	registryURL := parseHelmFlags().registryURL
+	if registryURL == "" {
+		return "", fmt.Errorf("could not extract repository from helm command arguments: registry URL not found")
+	}
+
+	return extractRepositoryNameFromURL(registryURL), nil
+}
+
+// extractRepositoryNameFromURL extracts the repository name from an OCI or HTTPS URL
+func extractRepositoryNameFromURL(repository string) string {
```

Comment:
```suggestion
func extractRepositoryNameFromURL(repositoryURL string) string {
```
it is very confusing to repositoryURL as repository